### PR TITLE
cmake msg files

### PR DESCRIPTION
**Pull request by bkueng**
Make sure cmake properly handles changes/adding/removing of .msg files.

### CODE REVIEW

File: msg/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+# OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+# AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+# LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+# ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+############################################################################
+
+set(msg_file_names
+	actuator_armed.msg
```
Comment (LorenzMeier) [inline msg/CMakeLists.txt]: Is there no way to pull this from the directory listing?

**Comment by LorenzMeier** [comment]
This change is great! It certainly was not a good thing to miss out on the dependency. Do we have to use a list of messages to get this?

**Comment by bkueng** [comment]
Yes we need an explicit list. I would prefer something autogenerated as well, but then we land back at using GLOB. cmake developers explicitly advice against using GLOB, because it misses dependency changes.

**Comment by LorenzMeier** [comment]
Rebased and applied.

**Pull request closed without merging**